packages/shared/__tests__/components/Input/inputPassword.test.jsx

```diff
@@ -21,18 +21,18 @@ const handleMouseDownPassword = (event) => {
 
 describe('Tests for the general input', () => {
   test('Test Input Password component render', () => {
-    const component = render(<InputPassword id="test" label="test" name="test" auto="user" />);
+    const component = render(<InputPassword id="test" label="test" name="test" auto="user" errorMessage="test" />);
```

Comment:
errorMessage value should be a constant due to you are repeating too many times

Reply:
i dont understand what do you mean by that, the errorMessage can change depending where its called

Reply:
Yeah, errorMessage always is test, so make constant which it's value is 'test'.
Something like this
```suggestion
    const messageError = "test"
    const component = render(<InputPassword id="test" label="test" name="test" auto="user" errorMessage={messageError} />);
```

Reply:
ok, now i get it, but wouldnt that be more work for the exact same result?

Reply:
No, because the code in one moment will need refactoring. Make repetitive things constants or function make it easy to refactor